app/generate.js

```diff
@@ -1,106 +1,166 @@
 import { template } from 'lodash';
-import { getData } from './data';
 import fs from 'fs';
-import { deleteFolderRecursive } from "./utils";
+import { getGraphData } from './data';
+import { deleteFolderRecursive } from './utils';
 
 function createDistFolder(name) {
-  const distName = './dist/' + name;
+  const distName = `./dist/${name}`;
   if (fs.existsSync(distName)) {
     deleteFolderRecursive(distName);
   }
-  fs.mkdirSync(distName, {recursive: true});
+  fs.mkdirSync(distName, { recursive: true });
 }
 
-function generateSections(theme, sections) {
-  let retArray = [];
-  const sectionTemplate = template(fs.readFileSync('./templates/' + theme + '/section.tpl.html', 'utf8'));
-  const imageTemplate = template(fs.readFileSync('./templates/' + theme + '/image.tpl.html', 'utf8'));
-  
-  for (let i = 0; i < sections.length; i++) {
-    let section = sections[i];
+function generateSections(theme, sections, imageOutput) {
+  const retArray = [];
+  const sectionTemplate = template(fs.readFileSync(`./templates/default/section.tpl.html`, 'utf8'));
+
+  for (let i = 0; i < sections.length; i += 1) {
+    const section = sections[i];
+
+    //  Send image url and id here
 
     if (section.image) {
-      section.imageHtml = imageTemplate(section.image);
+      const image = imageOutput.find((element) => {
+        if (element.id === section.image.id) {
+          return true;
+        }
+        return false;
+      });
+
+      section.imageHtml = image.html;
     } else {
-      section.imageHtml = "";
+      section.imageHtml = '';
     }
-    retArray.push(sectionTemplate(section));
-    //Does the same thing with header and footer templates, but with sections.
-    //Puts all the information in an array and saves it in sectionOutput'i'.
+
+    retArray.push({ html: sectionTemplate(section), id: section.id });
+    //  Does the same thing with header and footer templates, but with sections.
+    //  Puts all the information in an array and saves it in sectionOutput'i'.
```

Comment:
What do these comments give information about? Right now it's very unclear - either move them to a better place or remove them